pkgs/development/python-modules/fastf1/default.nix

```diff
@@ -0,0 +1,77 @@
+{
+  # lib & utils
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+  nix-update-script,
+
+  # build
+  hatchling,
+  hatch-vcs,
+
+  # deps
+  matplotlib,
+  numpy,
+  pandas,
+  python-dateutil,
+  rapidfuzz,
+  requests,
+  requests-cache,
+  scipy,
+  timple,
```

Comment:
It seen that `timple` does not exist on nixpkgs, so you will need to add it too (in a separate commit, preceding the one adding this package)